### PR TITLE
Create linux-recommendations.md

### PR DESCRIPTION
**Pull request by OxFred**
As requested during the Learnup, the documentation regarding Linux installation & security.

### CODE REVIEW

File: drafts/linux-recommendations.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+
+## How
+
+### Install Linux in a Virtual Machine (VM)
+
+Running Linux in a Virtual Machine is the best way to get a feel of what to expect.
+For some use cases, it could also be acceptable to operate long term by running Linux in a VM.
+But be aware from a security point of view, a VM and guest operating system will always somehow inherit the security of the host.
+Meaning that if your main host system is compromised, you should ultimately consider the guess insecure too. 
+
```
Comment (corpetty) [inline drafts/linux-recommendations.md:35]: guest compromise can be partially mitigated via disk encryption

File: drafts/linux-recommendations.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+### Buy a laptop with Linux pre-installed
+
+Some specialized computer vendors sell privacy-preserving laptops pre-installed with Linux. The two most popular ones being: 
+
+- System76 https://system76.com/ & its Pop OS! distribution
+- Purism Librem https://puri.sm/products/ & its PureOS distribution
+- Tuxedo https://www.tuxedocomputers.com/en
+
```
Comment (corpetty) [inline drafts/linux-recommendations.md:46]: maybe adding framework here? They're a new company that's pretty awesome:

https://www.youtube.com/watch?v=0rkTgPt3M4k

File: drafts/linux-recommendations.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+- For Fedora: Installation Destination screen > Encryption > ☑ Encrypt my data
+![](https://docs.fedoraproject.org/en-US/fedora/f34/install-guide/_images/anaconda/StorageSpoke.png)
+- For Ubuntu: Installation Type screen > Advanced features > ☑ Encrypt the new Ubuntu installation for security
+![](https://ubuntucommunity.s3-us-east-2.amazonaws.com/original/2X/4/46e9947d7b4ad3e96487d81cf61c327485d56b5a.png)
+
+#### Other Linux Distributions 
+
+Other popular Linux distributions includes:
+- Debian https://debian.org/
```
Comment (corpetty) [inline drafts/linux-recommendations.md:90]: I quite like PopOS for newb folks. default disk encryption, user friendly and clean interface, quality of life features with laptops and hybrid graphics drivers, based on Ubuntu but additional security checking for packages.

File: drafts/linux-recommendations.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+Depending on the types of updates, a computer restart may or may not be required to apply them.
+
+On Fedora to apply quickly security updates:
+`sudo dnf install -y --security`
+
+#### Common applications
+
+##### Web browsers
+
+Try to stick to Firefox or Chromium because they are usually provided, hardened & updated by the Linux distribution.
```
[the remark this answers is not in the document]
Comment (corpetty) [reply drafts/linux-recommendations.md:182]: opera isn't bad, and they're more crypto friendly and not based on chromium.

**Review by corpetty — CHANGES_REQUESTED** [review]
thoughts on us making a base image people can use to just flash? we pick all base options and software and create an ISO from it that people can use?

File: drafts/linux-recommendations.md (hunk 3 above)
Comment (OxFred) [reply drafts/linux-recommendations.md:90]: Sure of course

File: drafts/linux-recommendations.md (hunk 2 above)
Comment (OxFred) [reply drafts/linux-recommendations.md:46]: Sure, added

File: drafts/linux-recommendations.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (OxFred) [reply drafts/linux-recommendations.md:182]: I was big opera (mobile) user but they are now chromium based + still closed source...

File: drafts/linux-recommendations.md (hunk 1 above)
Comment (OxFred) [reply drafts/linux-recommendations.md:35]: Sure, I added it

**Comment by OxFred** [comment]
> thoughts on us making a base image people can use to just flash? we pick all base options and software and create an ISO from it that people can use?

This is something NixOS do very well.
For the Redhat family distributions there is https://weldr.io/lorax/ but then you have to keep it updated or automating the build...

**Pull request merged**